packages/e2e_tests/cypress.config.js

```diff
@@ -9,11 +9,12 @@ module.exports = defineConfig({
 			allureWriter(on, config);
 			return config;
 		},
-		specPattern: 'cypress/e2e/**/*.feature',
+		//specPattern: 'cypress/e2e/**/*.feature',
```

Comment:
Suggestion: Can you remove this comment or replace the specPattern with a correct glob pattern